examples/create_identity_provider/google/README.md

```diff
@@ -0,0 +1,37 @@
+# Google identity provider example
+
+Using Google as an identity provider allows any Google user to authenticate to your server.
+You can limit authentication to members of a specific hosted domain with the `hosted_domain` configuration attribute.
+
+This example shows how to create a Google identity provider.
+
+## Prerequisites
+
+### An OpenShift cluster
+
+This example assumes you have created a cluster via OCM, It can be via this terraform provider or by a different client.
+We will need the cluster name and some credentials.
+
+### Setting up your application in Google
+
+You will need a client ID/Secret of a registered Google project.
+The project must be configured with a redirect URI of `https://oauth-openshift.apps.<cluster-name>.<cluster-domain>/oauth2callback/<idp-provider-name>`.
+> **Note**: `<idp-provider-name>` is case-sensitive.
```

Comment:
Where do you set this?
Can you provide link/instructions?

Reply:
Added a link